core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -0,0 +1,50 @@
+
+// Notice!
+//
+// If you update this file, you *must* be sure to re-run
+//
+//  core/tools/ldml-const-builder/build.sh clean build run
+//
+// To update keyboardprocessor_ldml.h, and commit the result.
+//
+// It is not updated automatically.
+
+// namespace com.keyman.core.ldml {
```

Comment:
We'll need to figure this out at some point, so perhaps:
```suggestion
// TODO-LDML: namespace com.keyman.core.ldml {
```